.github/workflows/generate-template.yaml

```diff
@@ -65,6 +63,10 @@ jobs:
           flutter pub add --dev solid_lints
           flutter pub get
 
+      - name: Set dart sdk version
+        run: |
+          a=`sed '22!d' pubspec.yaml` && sed -i "s/$a/  sdk: '>=2.17.0 <4.0.0'/1" pubspec.yaml
```

Comment:
Ah, the good ol' bash magic.
You write it and it works, but then it breaks and nobody knows what it even does.

@solid-oleksandrhordiienko Could you please give some context on what we're trying to accomplish here?

Reply:
I see -- by default the template generates `sdk: '>=2.19.0 <3.0.0'` and we're trying to widen the constraints to be `'>=2.17.0 <4.0.0'`. But why do we need this?